packages/angular/src/side-panel/side-panel.component.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  Component,
+  EventEmitter,
+  HostBinding,
+  Input,
+  OnInit,
+  Output,
+  ViewEncapsulation,
+} from '@angular/core';
+import {
+  Close16,
+  ChevronLeft16,
+  ChevronRight16,
+  OpenPanelLeft16,
+  OpenPanelRight16,
+} from '@carbon/icons';
+import { IconService } from 'carbon-components-angular';
+
+/**
+ *
+ * [See demo](../../?path=/story/components-side-panel--basic)
```

Comment:
Why not the full URL?

Reply:
Cause it doesn't work on localhost and relative works anywhere.

Reply:
Oh nice, I thought the link is just there in the comments, didn't realize that it'll render somewhere